packages/katello/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -100,14 +99,17 @@ cp -r  $RPM_BUILD_DIR/%{gem_name}-%{version}/vendor/assets/fonts/*  %{buildroot}
 %{gem_instdir}/db
 %{gem_libdir}
 %{gem_instdir}/locale
-%{gem_instdir}/public
 %{gem_instdir}/script
 %{gem_instdir}/vendor
 %exclude %{gem_cache}
 %{gem_spec}
 %{foreman_bundlerd_plugin}
-
-%config(noreplace) /etc/redhat_access/config.yml
+%{foreman_apipie_cache_foreman}
+%{foreman_apipie_cache_plugin}
+%{foreman_assets_plugin}
+# The next line is foreman_assets_plugin if the gem name was insights
+%{gem_instdir}/public/assets/insights
```

Comment:
I think this is here because of https://github.com/redhataccess/foreman-plugin/tree/master/redhat-access/vendor/assets/images/insights. It might be the cause of your missing images that Rails can't find them. From `redhat-access/vendor/assets/stylesheets/telemetry/sat6.css`

```
.band.ansible{background:url(../insights/images/bg_ansible.jpg) no-repeat;
```

Perhaps the rails assets pipeline is not rewriting that properly? This is actually important because we send caching headers to indefinitely cache files so in a future upgrade users wouldn't pick up on changes. In the assets pipeline files are named with a hash in the filename to work around this. Because of this, I did want to properly compile them rather than move them around as it did before.

Reply:
so, what are the next steps?